rotkehlchen/tests/unit/decoders/test_arbitrum_one_bridge.py

```diff
@@ -302,6 +303,50 @@ def test_withdraw_erc20_from_arbitrum_one_to_ethereum(database, arbitrum_one_inq
     ]
 
 
+@pytest.mark.vcr()
+@pytest.mark.parametrize('arbitrum_one_accounts', [['0x9531C059098e3d194fF87FebB587aB07B30B1306']])
+def test_withdraw_dai_from_arbitrum_one_to_ethereum(database, arbitrum_one_inquirer, arbitrum_one_accounts):  # noqa: E501
+    """
+    Test that DAO withdrawals from arbitrum to L1 work fine. This is just to test that
```

Comment:
still thinking of DAOs huh?

```suggestion
    Test that DAI withdrawals from arbitrum to L1 work fine. This is just to test that
```